src/Tests/Tests/epv_test_validate_php_functions.php

```diff
@@ -230,22 +243,18 @@ private function parseNodes(array $nodes)
 	{
 		if (!($nodes[0] instanceof Namespace_))
 		{
-			$err = false;
 			foreach ($nodes as $node)
 			{
 				// Check if there is a class.
 				// If there is a class, there should be a namespace.
 				if ($node instanceof Class_ || $node instanceof InterfaceTest)
```

Comment:
Please add : 
```suggestion
				if ($node instanceof Class_ || $node instanceof InterfaceTest || $st instanceof Trait_)
```

for Fix #102